Blazor_Foundation_6_Test_Shared/Documentation/Containers/ResponsiveAccordionTabs.razor

```diff
@@ -1,7 +1,105 @@
 ﻿@page "/responsive-accordion-tabs";
 
-<h3>Responsive Accordion Tabs</h3>
+@using OpenCodeDev.Blazor.Foundation.Doc.Core.Components.Containers.Section;
+@using OpenCodeDev.Blazor.Foundation.Components.Controls;
+@using OpenCodeDev.Blazor.Foundation.Components.Containers;
+@using Microsoft.AspNetCore.Components.Forms;
 
-@code {
 
+
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Responsive Accordion Tabs</h2>
+        <p>
+            Added in 6.3.0, use the Markup of the Accordion or Tabs components to create Responsive Accordion Tabs.
+        </p>
+        <h2>Basics</h2>
+        <p>
+            Either the <NavLink href="/accordion">Accordion Markup</NavLink> or the <NavLink href="/tabs">Tabs Markup</NavLink>can be used to responsively switch between the two components at different breakpoints.
```

Comment:
Accordion Components and Tab Components

Space is missing after Tabs Markup

Make sure to show example of what you are talking about external controller with ids make sure that it works with Blazor Foudnation, some foundation features are not support so you have to test it and locate where it doesn't work so we can possible add support for it.